src/client/components/UsersListComponent/UsersList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './UsersListStyle.css';
+
+export default function Userslist(props) {
+  return (
+    <div className="container">
+      <div className="wraper">
+        <ul>
+          {props.people.map((people) => (
+            <li key={people}>{people}</li>
+          ))}
```

Comment:
Hopefully, JavaScript would be smart enough to avoid name collisions here but I would suggest renaming the variable to make it more clear :) Also why not call it `users` instead of `people`? :) 

And another thing when you provide the `key` it should be a string, if this is an object it will probably not work well. (just for later)

Iterating through `users` would be `user`, example:
```javascript
{props.users.map((user) => (
            <li key={user}>{user}</li>
          ))}
```